app/src/features/surveys/view/survey-animals/telemetry-device/TelemetryDeviceForm.tsx

```diff
@@ -20,27 +20,27 @@ const TelemetryDeviceForm = (props: ITelemetryDeviceFormProps) => {
   const { mode } = props;
 
   const telemetryApi = useTelemetryApi();
-  const { values, validateField } = useFormikContext<IAnimalTelemetryDevice>();
+  const { values } = useFormikContext<IAnimalTelemetryDevice>();
 
   const device = values;
 
   const { data: deviceDetails, refresh } = useDataLoader(() =>
-    telemetryApi.devices.getDeviceDetails(Number(device.device_id))
+    telemetryApi.devices.getDeviceDetails(Number(device.device_id), device.device_make)
   );
 
-  const validateDeviceMake = async (value: number | '') => {
-    const deviceMake = deviceDetails?.device?.device_make;
-    if (device.device_id && deviceMake && deviceMake !== value && mode === ANIMAL_FORM_MODE.ADD) {
-      return `The current make for this device is ${deviceMake}`;
-    }
-  };
-
+  //TODO remove?
```

Comment:
we can probably remove this, looks like the old variable isn't being used anymore and if we really need this code it is in the repo already